index.html

```diff
@@ -35,12 +35,235 @@ <h2 class="headline-title2">I'm a Software Developer!</h2>
+        </section>
+
+        <section class="center-content">
+            <article class="content-desc">
+                <h2>Profesional Art <br />Printing Data</h2>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. Has been the industry's standard.
+                </p>
+                <ul>
+                    <li>
+                        <a href="#">HTML</a>
+                    </li>
+                    <li>
+                        <a href="#">BOOTSTRAP</a>
+                    </li>
+                    <li>
+                        <a href="#">RUBY</a>
+                    </li>
+                </ul>
+                <button type="button">See project</button>
+            </article>
+        </section>
+
+        <section class="about">
+            <article class="about-content">
+                <div class="about-header">
+                    <h2>About Me</h2>
+                    <p>
+                        Hello I’m a software developer! I can help you build a product,
+                        feature or website Look through some of my work and experience! If
+                        you like what you see and have a project you need coded, don’t
+                        hesitate to contact me.
+                    </p>
+                    <button type="button">Get my resume</button>
+                </div>
+                <div class="about-footer">
+                    <hr />
+                    <ul class="major-menu">
+                        <li>
+                            <div class="lang-first">
+                                <img src="images/rhombus.png" width="18px" height="18px" alt="" />
+                                <h2>Language</h2>
+                            </div>
+                            <ul class="minor-menu">
+                                <li class="jss">Javascript</li>
+                                <li class="jss">Ruby</li>
+                                <li class="jss">Html</li>
+                                <li class="jss">Css</li>
+                            </ul>
+                        </li>
+                        <li>
+                            <div class="lang-sec">
+                                <img src="icons/rectangle.png" width="18px" height="18px" alt="" />
+                                <h2>Frameworks</h2>
+                            </div>
+                            <ul class="minor-menu">
+                                <li class="jss">Bootstrap</li>
+                                <li class="jss">Ruby on Rails</li>
+                                <li class="jss">RSPec</li>
+                                <li class="jss">CapyBara</li>
+                                <li class="jss">Selenium</li>
+                            </ul>
+                        </li>
+                        <li>
+                            <div class="lang-sec">
```

Comment:
- [ ] I really appreciate your effort in trying to match your design with the design given 👏. In addition, great job you have used list tags here 👏. What I would like you to improve on is that, as you see the technology categories("Languages", "Frameworks", "Skills") are list items and it is right but their belongings(for example; Languages list contains "Ruby", "Javascript", etc ) should be nested inside them. Let me show you what I would like you to do by example;
```html
<ul>
     <li>Languages
            <ul>
                 <li>Javascript</li>
                <li>Ruby</li>
                <li>Html</li>
               <li>CSS</li>
            </ul>
     </li>
<li>Frameworks
            <ul>
                 <li>Bootstrap</li>
                <li>Ruby on Rails</li>
                <li>RSPec</li>
               <li>CapyBara</li>
               <li>Selenium</li>
            </ul>
     </li>
<li>Skills
            <ul>
                <li>Codekit</li>
                 <li>Github</li>
                <li>Codepen</li>
               <li>Gitlab</li>
               <li>Terminal</li>
            </ul>
     </li>
</ul>
```
This is what I would like you to follow because it is **more logical** and it is among the requirements. It is referred to as **Nesting list into another list**. For more information, please read this [documentation](https://www.webucator.com/article/how-to-properly-nest-lists-in-html/) 💡 😃